rosdep/python.yaml

```
@@ -1798,6 +1798,15 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python-flask-sqlalchemy:
```

This should make the Ubuntu package name, so should be:

```suggestion
python3-flask-sqlalchemy:
```